inspirehep/modules/orcid/tasks.py

```diff
@@ -93,35 +89,63 @@ def _link_user_and_token(user, name, orcid, token):
         # User already has their ORCID linked
         pass
 
-    # Check whether there are already tokens associated with this
-    # ORCID identifier.
-    tokens = RemoteToken.query.join(RemoteAccount).join(User)\
-        .join(UserIdentity).filter(UserIdentity.id == orcid).all()
+    # Search for existent tokens.
```

Comment:
`existent`->`existing` (also in the following)

Reply:
Done, in all occurrences